data/strings/strings.txt

```diff
@@ -23516,83 +23542,94 @@
+    ko:other = %d 개의 파일이 발견되었습니다. 변환 후 확인할 수 있습니다.
     lv:one = Lietotne atrada %d jaunu datni. Varat to apskatīt pēc konvertācijas.
     lv:other = Lietotne atrada %d jaunas datnes. Varat tās apskatīt pēc konvertācijas.
     lv:zero = Lietotne atrada %d jaunu datņu. Varat tās apskatīt pēc konvertācijas.
-    mr:one = %d फाईल सापडली. रूपांतरानंतर तुम्ही ती पाहू शकता.
-    mr:other = %d फाईली सापडल्या. रूपांतरानंतर तुम्ही त्या पाहू शकता.
-    nb:one = %d filen ble funnet. Du vil se dem etter konverteringen.
+    mr:one = %d फाइल सापडली. रूपांतरानंतर तुम्ही ती पाहू शकता.
+    mr:other = %d फाइल सापडल्या. रूपांतरानंतर तुम्ही त्या पाहू शकता.
+    nb:one = %d fil funnet. Du vil se den etter konverteringen.
     nb:other = %d filer funnet. Du vil se dem etter konverteringen.
     nl:one = %d bestand gevonden. Je ziet het na het converteren.
-    nl:other = Er zijn %d bestanden gevonden. Je zult ze na de conversie zien.
-    pl:one = Znaleziono %d plik. Zobaczysz to po konwersji.
+    nl:other = %d bestanden gevonden. Je zult ze na de conversie zien.
+    pl:few = Znaleziono %d pliki. Zobaczysz je po konwersji.
+    pl:one = Znaleziono %d plik. Zobaczysz go po konwersji.
     pl:other = Znaleziono %d plików. Zobaczysz je po konwersji.
-    pl:two = Znaleziono %d pliki. Zobaczysz je po konwersji.
-    pt-BR:one = %d arquivo foi encontrado. Você vai ver depois da conversão.
-    pt-BR:other = %d arquivos foram encontrados. Você os verá depois da conversão.
+    pt-BR:one = %d arquivo encontrado. Você verá depois da conversão.
+    pt-BR:other = %d arquivos encontrados. Você os verá depois da conversão.
     pt:one = %d ficheiro encontrado. Pode vê-lo depois da conversão.
     pt:other = %d ficheiros encontrados. Pode vê-los depois da conversão.
-    ro:one = A fost găsit %d fișier. Îl vei vedea după conversiune.
-    ro:other = Au fost găsite %d fișiere. Le vei vedea după conversiune.
+    ro:one = A fost găsit %d fișier. Îl vei vedea după conversie.
+    ro:other = Au fost găsite %d fișiere. Le vei vedea după conversie.
     ru:few = %d файла были найдены. Вы увидите их после конвертации.
     ru:one = %d файл был найден. Вы увидите его после конвертации.
     ru:other = %d файлов было найдено. Вы увидите их после конвертации.
     sk:few = Boli nájdené %d súbory. Uvidíte ich po konverzii.
-    sk:one = Bol nájdený %d súbor. Uvidíte to po konverzii.
+    sk:one = Bol nájdený %d súbor. Uvidíte ho po konverzii.
     sk:other = Bolo nájdených %d súborov. Uvidíte ich po konverzii.
-    sr:few = %d фајла је пронађено. Можете да их погледате након конверзије.
-    sr:one = %d фајл је пронађен. Можете да га погледате након конверзије.
-    sr:other = %d фајлова је пронађено. Можете да их погледате након конверзије.
+    sr:few = %d фајла су пронађена. Можете их погледати након конверзије.
+    sr:one = %d фајл је пронађен. Можете га погледати након конверзије.
+    sr:other = %d фајлова је пронађено. Можете их погледати након конверзије.
```

Comment:
`sr` was added just recently by humans.
It looks like the AI changed e.g. politeness, but its not related to plurals.